src/app/containers/SettingsPage/index.js

```diff
@@ -0,0 +1,139 @@
+import React, { useEffect } from 'react';
+import PropTypes from 'prop-types';
+
+// APIs
+import { apiGetMyUserInfo, apiUpdateUserInfo } from 'src/api/user';
+
+// HOC
+import { withRouter } from 'react-router-dom';
+import withUserLogin from 'src/shared/hoc/withUserLogin';
+import { UserInfoContext } from 'src/shared/context/UserInfo';
+
+// Components
+import Loading from '../../common-ui/Loading';
+
+// UI
+import * as MainPanel from '../../common-ui/MainPanel';
+import * as Form from '../../common-ui/Form';
+import { PrimaryButton } from '../../common-ui/Button';
+import { Helmet } from 'react-helmet';
+import { ErrorPopup, SuccessPopup } from '../../common-ui/Popup';
+
+// Constants and utils
+import { API_PROGRESS } from 'src/shared/constants';
+import { validate } from './validators';
+import { ROUTE_LOGIN } from '../../routes/constants';
+
+const labels = {
+  fullname: 'Họ và tên',
+  school: 'Trường học',
+  email: 'Email liên lạc',
+};
+
+function SettingsPage({ history, location }) {
+  const { userInfo } = React.useContext(UserInfoContext);
+  const [values, setValues] = React.useState({});
+  useEffect(() => {
+    (async () => {
+      const { full_name: fullname, school_name: school, email } = await apiGetMyUserInfo(
+        userInfo.token,
+      );
+      setValues({ fullname, school, email });
+    })();
+  }, []);
```

Comment:
check if we need to call API or it's already in `userInfo ` of UserInfoContext. Because we save it in context already